atd-vze/src/views/Crashes/Crash.js

```diff
@@ -55,6 +55,16 @@ const calculateYearsLifeLost = people => {
 };
 
 function Crash(props) {
+  const [isCOA, setIsCOA] = useState(false);
+  useEffect(() => {
+    const storedEmail = window.localStorage.getItem("hasura_user_email");
+    if (storedEmail) {
+      if (storedEmail.toLowerCase().endsWith("@austintexas.gov")) {
+        setIsCOA(true);
```

Comment:
To get this state set to `true`, you have to have an austintexas.gov address. Otherwise, this state variable `isCOA` will continue to have the default value of `false`. To test this PR and simulate not having an `austintexas.gov` address, just comment out this line.

Reply:
@frankhereford I'm taking a look at this, and I'm pretty sure that we can get away without the `useEffect` and the `isCOA` state since `isCOA` can be derived directly from `user` below.

Reply:
Mike and I looked at this, and I believe that the useEffect is needed to cause the component to re-render when the `user` authentication data is loaded.

Reply:
Further @mddilley stuck with me and helped me realize that he had been absolutely correct from the get-go. Thank you!